pkg/aexec/cmd.go

```diff
@@ -0,0 +1,111 @@
+package aexec
```

Comment:
what does the "a" in "aexec" stand for? "async"?

Reply:
yes, any other suggestions ? i was also thinking about putting in the `cmd/common` file